src/components/user/schedule/Schedule.jsx

```diff
@@ -19,8 +19,10 @@ const Schedule = ({ eventList }) => {
 
   return (
     <>
-      <Toolbar onFilterChange={filterChange} />
-      <div className="sticky top-0 z-50 flex w-full bg-gray-100 text-white">
+      <div className="sticky top-0 z-[1000] bg-gray-100">
```

Comment:
this div can be integrated directly into the toolbar since the toolbar is made just for the schedule component

Reply:
ohh okay i see. thank you.